src/test/java/org/jabref/gui/search/ContainsAndRegexBasedSearchRuleDescriberTest.java

```diff
@@ -0,0 +1,80 @@
+package org.jabref.gui.search;
+
+import java.util.Arrays;
+import java.util.List;
+
+import javafx.scene.text.Text;
+import javafx.scene.text.TextFlow;
+
+import org.jabref.gui.search.rules.describer.ContainsAndRegexBasedSearchRuleDescriber;
+import org.jabref.gui.util.TextUtil;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertTrue;
+
+public class ContainsAndRegexBasedSearchRuleDescriberTest {
+
+    private boolean checkIfDescriptionEqualsExpectedTexts(TextFlow description, List<Text> expectedTexts) {
+        if (expectedTexts.size() != description.getChildren().size())
+            return false;
+        Text expectedText;
+        for (int i = 0; i < expectedTexts.size(); i++) {
```

Comment:
Better convert the two lists to sets and use streams 
https://stackoverflow.com/a/30346012 (second approach)
Furthermore, as this method is also used in the other test it makes sense to extract it as one TestUtil method, to a new or an existing "testutil" class. I don' kntpw

Reply:
but the order is important for the test so I dont think a set is helpful here. Every item has to be in the right position with the correct properties.

Reply:
Yeah, okay, then stick with this. But please extract it to a new class so that it can be reused across the tests